robot_sort/robot_sort.py

```diff
@@ -92,11 +92,42 @@ def light_is_on(self):
         """
         return self._light == "ON"
 
+    def findSmallest(self):
```

Comment:
Stretch goals look good. All your times are coming in well under what the tests are expecting 👍 Great usage of your helper functions to optimize the runtime _and_ shorten up the actual `sort()` function Julie.